docs/contributing/installing.md

```diff
@@ -0,0 +1,52 @@
+---
+id: install
+title: Contributing
+sidebar_label: Install
+---
+
+# Install
+
+## Prerequisits
+
+### Setup a development environment
+
+1. Install the latest stable versions of [Git](https://git-scm.com/download).
+1. Configure git user credentials for [every respository](https://help.github.com/articles/setting-your-username-in-git/#setting-your-git-username-for-every-repository-on-your-computer) or [single repository](https://help.github.com/articles/setting-your-username-in-git/#setting-your-git-username-for-a-single-repository)
+1. Install the LTS (Long Term Support) version of [Node](https://nodejs.org/en).
```

Comment:
FYI - We are not on LTS, we are on 10 something

Reply:
Or at least > 9 I believe

Reply:
@chrisdholt : LTS is 10.14.1 today.  My point is install the latest stable version.  Unless our code requires a specific version?  I don't believe it does?  What would you like to say in this section or should we just remove it?

Reply:
Section removed